backend/src/test/java/hanglog/trip/restdocs/RestDocsConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package hanglog.trip.restdocs;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.restdocs.mockmvc.MockMvcRestDocumentation;
+import org.springframework.restdocs.mockmvc.RestDocumentationResultHandler;
+import org.springframework.restdocs.operation.preprocess.Preprocessors;
+import org.springframework.restdocs.snippet.Attributes.Attribute;
+
+
+@Configuration
+public class RestDocsConfiguration {
+
+    public static final Attribute field(final String key, final String value) {
+        return new Attribute(key, value);
+    }
```

Comment:
<img width="630" alt="image" src="https://github.com/woowacourse-teams/2023-hang-log/assets/77482065/c2629813-38e5-467f-bd6a-427d1c6c580c">


```suggestion
    public static Attribute field(final String key, final String value) {
        return new Attribute(key, value);
    }
```